app/controllers/dashboard/flickr_controller.rb

```diff
@@ -0,0 +1,14 @@
+require 'flickraw'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.